testing/zerotier-one/zerotier-one.initd

```diff
@@ -1,11 +1,14 @@
 #!/sbin/openrc-run
 
-depend() {
-    provide net
-    use network
-    before firewall netmount
-}
-
+name=zerotier-one
 command="/usr/sbin/zerotier-one"
+command_args="$zerotier_opts"
```

Comment:
It’s fine, we can always add `command_args="--some-args --more-args $command_args"`.